build/templates/_attributes.py.mako

```diff
@@ -104,6 +104,29 @@ class AttributeEnum(object):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeBooleanEnum(object):
+    def __init__(
+        self,
+        underlying_attribute_meta_class,
+        attribute_id,
+        enum_value_to_bool_map
+    ):
+        self._underlying_attribute = underlying_attribute_meta_class(attribute_id)
+        self._attribute_id = attribute_id
```

Comment:
seems you're storing things that aren't needed

Reply:
This class has been updated in my latest commits.